modules/lead/product-jenkins/variables.tf

```diff
@@ -34,3 +34,6 @@ variable "pipelines" {
 
 variable "vault_namespace" {}
 variable "vault_root_token_secret" {}
+variable "vault_adress" {
+  default = "http://vault.${var.vault_namespace}.svc.cluster.local"
```

Comment:
you cannot use a variable as part of the default value for another variable

Reply:
my bad, thought you could